superset/security/api.py

```diff
@@ -160,3 +163,33 @@ def guest_token(self) -> Response:
             return self.response_400(message=error.message)
         except ValidationError as error:
             return self.response_400(message=error.messages)
+
+    @expose("/custom_login/", methods=["GET"])
+    @event_logger.log_this
+    @safe
+    @permission_name("read")
+    def custom_login(self) -> Response:
+        token_val = request.headers.get("Authorization")
+
+        headers = {"Authorization": token_val, "accept": "application/json"}
+
+        response = requests.get(
+            "https://dev-api.khinternal.net/platform-rbac-stage/v1/users/me?expand=organizations.accounts",
```

Comment:
@pradasouvanlasy how to un-hardcode this?

Reply:
I would expect something like adding a new `PLATFORM_RBAC_BASE_URL` variable in `superset_config.py` and reading it back there.

That would imply updating the Helm chart in the `superset-provisioning` repo in order to add that new variable as well.